### ISSUE TITLE
document common random numbers

### ISSUE THREAD

**Issue by adolgert** [adolgert/CompetingClocks.jl#49]
There isn't much on usage of common random numbers.

**Comment by slwu89** [comment]
What is the simplest possible example we could use to actually show variance reduction in practice of a crude Monte Carlo vs. Monte Carlo with CRN? It would be nice for a user to actually see this in practice.

**Comment by adolgert** [comment]
Yup. That's a great idea. I just redid the way the CRN works. Let me think about an example. Can we make that a different PR, so we can merge this one after yours?

**Comment by slwu89** [comment]
Yeah of course, let's work on this in a different PR.

**Comment by adolgert** [comment]
There is now a demo for common random numbers, so yay.
